### ISSUE TITLE
New classes

### ISSUE THREAD

**Issue by koljada** [stephansturges/WALDO#25]
Any plans to support more classes? In particular `train` class would be very useful for me.
Or maybe you give me a hint how can I extend your model?

**Comment by stephansturges** [comment]
You can fine-tune the models on your own data and include trains :) That would be the best way to do it.
Alternatively you could share with me your data that includes trains (with annotations) and I can integrate it into the training set for the next version, but bear in mind you will need a LOT of trains to make it work with any level of accuracy (at least 1000 annotations, ideally more like 10000).

**Comment by koljada** [comment]
@stephansturges Thanks for your reply
